rules/windows/process_creation/proc_creation_win_iis_appcmd_list_creds.yml

```diff
@@ -0,0 +1,31 @@
+title: Retrieve Microsoft IIS Service Account Creds Using Appcmd
+id: d65a2d9d-31c8-414b-8492-bcfde9b18ed0
+status: experimental
+description: The list and the config command within appcmd.exe leveraged by an adversary to obtain the username and password of the service account that is configured to run the application.
+references:
+    - https://twitter.com/0gtweet/status/1588815661085917186?cxt=HHwWhIDUyaDbzYwsAAAA
+author: Janantha Marasinghe
+date: 2022/11/06
+tags:
+    - attack.credential_access
+    - attack.t1552
+logsource:
+    category: process_creation
+    product: windows
+detection:
+    selection_name:
+        Image|endswith: '\appcmd.exe'
+    selection_list:
+        CommandLine|contains:
+            - ' /@t:* '
+            - ' /@text:* '
+            - ' /text:* '
+            - ' /config '
```

Comment:
Here we want to find the `*` . It need to be [escaped](https://github.com/SigmaHQ/sigma-specification/blob/main/Sigma_1_0_1.md#escaping)

They can be use at the end of the CommanLine like in the ref, so no space after.